clash-lib/src/Clash/Normalize/Strategy.hs

```diff
@@ -74,25 +74,25 @@ constantPropagation =
   dec >->
   conSpec
   where
-    etaTL              = apply "etaTL" etaExpansionTL !-> topdownR (apply "applicationPropagation" appPropFast)
+    etaTL              = apply "etaTL" etaExpansionTL !-> topdownR (apply "applicationPropagation" appProp)
```

Comment:
Can we get an `apply$transformation` for every transformation? I.e., the module exporting `appProp` would also export `applyAppProp`. Names go out of sync easily it seems, and its a lot of repetition in this file.